core/src/core/block.rs

```diff
@@ -261,6 +262,10 @@ impl PMMRable for BlockHeader {
 			is_secondary: self.pow.is_secondary(),
 		}
 	}
+
+	fn elmt_size() -> Option<u16> {
+		Some(HeaderEntry::LEN)
```

Comment:
Is the const HeaderEntry ::LEN used anywhere else? If not doesn't moving it inside the `elmt_size`makes sense?